### ISSUE TITLE
deploy react-todo - showing error message

### ISSUE THREAD

**Issue by williamyinssw** [SharePoint/sp-dev-fx-webparts#110]
#### Category
- [x] Question
- [x] Bug
- [ ] Enhancement

#### Expected or Desired Behavior
I tried to deploy react-todo to my developer tenant, and I was expect to see the todo list.
(I have done this a few month ago, and it worked fine)

#### Observed Behavior
but I got the below error:
[SPWebPartErrorCode.ScriptLoadError]:: Unable to load web part WebPart.null.g_64f30467_66e8_428d_90ee_a1bf17199d4e,Error: ***Failed to load component "2625c2e1-aa64-4b86-b922-d8190661d9f9" (null). Original error: Error loading https://component-id.invalid/2625c2e1-aa64-4b86-b922-d8190661d9f9_1.0.0 script resources due to: undefined. CALLSTACK:: Error at t [as constructor] (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-loader_en-us_4ec28636b0f9b465bd02537852aebf16.js:57:16731) at new t (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:47:22096) at Function.t.create (https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:47:22509) at https://spoprod-a.akamaihd.net/files/sp-client-prod_2017-01-16.005/sp-webpart-base_en-us_8d0a284d4d0d8e404879f4b5fcc7cc99.js:48:627

#### Steps to Reproduce
Basically, I followed the instruction to deploy to a developer site, then I got the error, there could be some wrong configuration I made.

**Comment by davidhartman** [comment]
@williamyinssw correct me if I'm wrong but it looks like you are attempting to load this webpart on the workbench in you tenant at /_layouts/15/workbench.aspx? If so that is pulling an updated version of the workbench which can cause webparts that have not been updated to break. So the author of the webpart probably hasn't had time to update the code to rc0.
